index.html

```diff
@@ -247,7 +247,7 @@ <h1>04</h1>
 								<div class="desc">
 									<h4>File Manager & Network Functionality</h4>
 									<p>
-										Archcraft has pre-configured Network-Manager And provides you the full file manager functionalities like Display and access other partitions, removable media (e.g. USB sticks, optical discs, and digital cameras), networking / shared networks, thumbnailing, Archive and extract compressed files, etc. Well sounds like no big deal, maybe for a newbie arch user.
+										Archcraft has pre-configured Network-Manager. It also provides you the full file manager functionalities like display and access other partitions, removable media (e.g. USB sticks, optical discs, and digital cameras), networking / shared networks, thumbnailing, archive tools (i.e. extracting compressed files), etc. Everything needed for a complete and usable daily driver.
```

Comment:
displaying, accessing, archiving tools?

Reply:
Yeah I didn't know

Reply:
I also thought it weird to mention display in file manager functionality. :thinking: Any thoughts?

Reply:
I guess it’s meant to be together with *accessing other partitions*.